media.css

```diff
@@ -0,0 +1,117 @@
+@media (max-width: 1550px) {
+  
+
+  
+  .center__v {
+    
+  }
+}
+
+  
+@media (max-width: 1453px) {
+  .middle__title {
+    right: 0px;
+    top:150px;
+  }
+
+  .content {
+    left: 0;
+    bottom: 0;
+    padding-left: 500px;
+    padding-right: 500px;
+  }
+
+  .container {
+  max-width: 100%;
+  bottom: 0px;
+  top: 900px;
+  }
+  .underside {
+  bottom: 1000px;
+  left: 0px;
+  }
+
+  .other {
+    flex-wrap: wrap;
+  }
+}
+
+
+@media (max-width: 1335px) {
+  
+  .menu__link_modelS {
+		display: none;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla3/pullRequest?prid=6027752)